src/test/java/core/basesyntax/services/FruitShopServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.services;
+
+import core.basesyntax.services.operations.Buy;
+import core.basesyntax.services.operations.Operable;
+import core.basesyntax.services.operations.Return;
+import core.basesyntax.services.operations.Supply;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitShopServiceTest {
+    private static final String FILE_PATH
+            = "src/test/resources/testFile.txt";
+    private static final String RESULT_FILE_PATH
+            = "src/test/resources/resultedTestFile.txt";
+    private static final String RETURN = "r";
+    private static final String SUPPLY = "s";
+    private static final String BUY = "b";
+    private static FruitShopService shopService;
+    private static Map<String, Operable> operationsMap;
+
+    @Before
+    public void before(){
+        shopService = new FruitShopService();
```

Comment:
FruitShopService doesn't have any fields, why we need to initialize it once again before each test ?

Reply:
changed to ```@BeforeClass```